.github/workflows/tests.yml

```diff
@@ -25,3 +25,9 @@ jobs:
       - name: Run Unit Tests
         run: python3 -m unittest discover
         working-directory: ./CMR/python
+      #- name: Package
```

Comment:
Let's delete commented out sections

Reply:
This file has a merge conflict, I'll do it then (actually I think master version will bring this code in)